CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
+## [2.3.5] - 2021-08-31
```

Comment:
Did you bump the version on version.rb?

Reply:
oh my gosh I totally forgot to do that 🤦‍♀️  - good spot, will do now